Client/Frontend/Browser/TabContentScript.swift

```diff
@@ -0,0 +1,76 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import Foundation
+import WebKit
+import BraveShared
+import Shared
+
+private let log = Logger.browserLogger
+
+extension TabContentScriptLoader {
+  static var uniqueID: String {
+    UUID().uuidString.replacingOccurrences(of: "-", with: "")
+  }
+  
+  static var messageUUID: String {
```

Comment:
Can you add documentation for `messageUUID` and `uniqueID`? So we know what they do more easily